core/services/form/meta/InputOptions.php

```diff
@@ -85,14 +84,16 @@ public function options(): array
 
 
     /**
-     * @param int|float|string $option_value
-     * @param int|float|string $display_text
+     * @param array $option
      */
-    public function addOption($option_value, $display_text): void
+    public function addOption($option): void
```

Comment:
plz type hint for parameters

```suggestion
    public function addOption(array $option): void
```

Reply:
Done, thanks, need to make that a habit 😄